focus-ios/Blockzilla/Utilities/URIFixup.swift

```diff
@@ -15,6 +15,14 @@ class URIFixup {
             return nil
         }
 
+        // Check if the URL string does not start with "file://". 
+        // If it starts with "file://", return nil, indicating that this function
+        // does not handle local file URLs, resulting in a google search
```

Comment:
```suggestion
        // does not handle local file URLs, resulting in a search
```

Removed google as a user might have another search engine selected